### ISSUE TITLE
虛擬環境安裝無法成功>

### ISSUE THREAD

**Issue by KKalleKK** [arleigh418/python-and-Taiwan-stock-market#22]

**Comment by arleigh418** [comment]
hi您好，很高興您提問.
請問您測試裡面的env是自己創建的嗎？ 還是下指令創建的？
我建議是把您手動創建的env資料夾砍掉，在測試那一層資料夾中下語法.
您無須自行手動創建資料夾，創建虛擬環境的語法會自動幫您產生虛擬環計用的資料夾.

**Comment by KKalleKK** [comment]
原本的是自己建立的，我將它刪掉後在下指令好像還是無法建立。由於在windowns 的cmd下指令好像都不會有甚麼反映，不知道是不是應該在其他地方下還是有其他的問題呢?
![image](https://user-images.githubusercontent.com/104818077/166611727-1aeaf635-6bec-4ffb-baf8-e56defcf904a.png)

**Comment by arleigh418** [comment]
hi您好, 這的確是很奇怪的現象,通常創建失敗都會有錯誤訊息,換個地方創建看看並換個虛擬環境名稱呢？
e.g. 在d:/底下直接下創建的語法,然後換一個虛擬環境的名稱,如： python -m venv test_env
看一下在d:/底下會不會有叫做test_env的虛擬環境.

**Comment by KKalleKK** [comment]
經過測試過好像還是不行~想確定是對windowns 的cmdv下指令嗎?

**Comment by arleigh418** [comment]
是的,確定是.通常造成這類型的問題多與環境有關係. e.g. 當時安裝python時可能未將python設定至path中,亦或者是電腦曾經有多個python環境沒有清理乾淨.

通常最直接簡單的solution有2:
1. 清理環境,將所有跟python有關的東西都刪掉,包含anaconda（如果有）
2. 如果第一點不太方便的話,可以改用virtualenv, 如果單看基本的虛擬環境套件管理的話, 他的用法與venv create的虛擬環境是極為類似的.應該不會有太大的學習成本.

**Comment by KKalleKK** [comment]
感謝您。瞭解了。
